backend/inspirehep/utils.py

```diff
@@ -85,6 +85,33 @@ def get_failure_message_for_index_record(task_name, exception, **kwargs):
     return format_failure_message_for_single(task_name, exception, kwargs.get("uuid"))
 
 
+def get_failure_message_for_disambiguate_records(
+    task_name, exception, uuid=None, record_version_id=None, **kwargs
+):
+    if isinstance(uuid, list):
+        uuid = [
+            f"{u} (version {rec_id})"
+            for u, rec_id in zip(uuid, record_version_id, strict=False)
```

Comment:
Here if record_version_id is None, the zip will return empty list. you can check that before for safety. Also if record_version_id is None, it will throw an error. Right now it is not the case but better to make it safer.